multisense_ros/cfg/multisense.cfg

```diff
@@ -284,6 +285,13 @@ for cfg in sensorConfigList:
 
     gen.add("max_point_cloud_range", double_t, 0, "max point cloud range", 15.0, 0.0, 100.0)
 
+    gen.add("origin_from_camera_position_x_m", double_t, 0, "Origin from camera extrinsics transform x value (m)", 0.0, -100.0, 100.0)
```

Comment:
I like the name much better. Just to be clear this take points in the camera frame and transforms them into the origin frame?

Reply:
That's right!